tests/e2e_test/utils_test.go

```diff
@@ -378,3 +407,40 @@ func CheckKymaModuleIsInState(ctx context.Context,
 	}
 	return fmt.Errorf("error checking kyma module state: %w", errModuleNotFound)
 }
+
+func CheckSampleCRIsInState(ctx context.Context, name, namespace string, clnt client.Client,
+	expectedState string,
+) error {
+	return CRIsInState(ctx,
+		"operator.kyma-project.io", "v1alpha1", "Sample",
+		name, namespace,
+		[]string{"status", "state"},
+		clnt,
+		expectedState)
+}
+
+func SampleCRNoDeletionTimeStampSet(ctx context.Context, name, namespace string, clnt client.Client) error {
+	_, exists, err := GetDeletionTimeStamp(ctx, "operator.kyma-project.io", "v1alpha1",
```

Comment:
Is there any purpose return this deletiontimestamp? no real usage

Reply:
Because I implemented it in the utils packge adn can be reused in future, if you think that is not neccessary, then I can remove it

Reply:
let's remove it, two reasons:
1. to check deletiontimestamp is not so usual, normally just need to verify if it exits.
2. avoid overrengineer, when we need, we can refactor this later.